browser.js

```diff
@@ -162,7 +162,6 @@ pino.levels = {
 }
 
 pino.stdSerializers = stdSerializers
-pino.symbols = require('./lib/symbols')
```

Comment:
I have a fear that this makes this change semver-major.
I would prefer it not to be.

Reply:
yep. But I think this is a case of semver purity vs practicality. polyfilling symbols is extra overhead and not fully possible (e.g. typeof symbol). There is only one use case, and it's technically server side - so I think an argument can be made in this case for considering the breaking of browser compat a bug, and removing this to be the solution to that bug. Since  this change along with the hapi-pino change will literally affect no one, as there are no other cases for pino.symbols in the browser.